frontend/src/pages/Overview.vue

```diff
@@ -1,15 +1,24 @@
 <template>
   <div class="overview">
-    <community-news />
+    <BCol>
+      <BRow>
+        <card-circles />
+      </BRow>
+      <BRow>
+        <card-user-search />
+      </BRow>
+    </BCol>
```

Comment:
Please check here if gms and humhub are allowed, to prevent unnecessary loading of components, if not needed. And especially unnecessary triggering the graphql requests when the backend will surly answer with an error. I have tried it, if I disable both in config and open the frontend in browser, it will issue two queries for one for humhub authentication, one for gms authentication.

Reply:
That's a good idea and now with the wellknown controlling it will be hopefully easier to check this here in the overview.vue...